### PR TITLE
Changed demisto to Cortex Xsoar in docs

### PR DESCRIPTION
**Pull request by yaakovpraisler**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Fixed 'Demisto' to 'Cortex XSOAR' in README and description.

## Minimum version of Cortex XSOAR
- [ ] 5.5.0
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by glicht** [comment]
@yaakovpraisler please provide information how you performed the change

**Comment by yaakovpraisler** [comment]
> @yaakovpraisler please provide information how you performed the change

When I added the validation that 'Demisto' would not appear in the Docs and descriptions (https://github.com/demisto/demisto-sdk/pull/1291), I got in the build validation the list of all files that contain the demisto word. So I go through all the files and when there is a reference to the Demisto product I change it to Cortex XSOAR.

File: Packs/CommonScripts/Scripts/script-SetByIncidentId_README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ Sets a value into the context with the given context key. By default this will n
 | **Name** | **Description** |
 | --- | --- |
 | Script Type | python |
-| Tags | DemistoAPI |
+| Tags | XSOARAPI |
```
Comment (yaakovi) [inline Packs/CommonScripts/Scripts/script-SetByIncidentId_README.md:9]: revert

File: Packs/CortexDataLake/Integrations/CortexDataLake/README.md [hunk 2]
```diff
@@ -893,7 +893,7 @@ Searches the URL table
 | limit | The number of logs to return. Default is 5. | Optional | 
 | ip | IP address. Enter an IP address or an array of IP addresses for which to search, for example 1.1.1.1,2.2.2.2. | Optional | 
 | port | Port utilized by the session. Enter a port or array of ports to search. | Optional | 
-| url | This argument allows to perform a LIKE search of the specified values on the Url and Uri fields An example value will be paloaltonetworks.com,demisto which will provide results like https://apps.paloaltonetworks.com and https://demisto.com | Optional | 
+| url | This argument allows to perform a LIKE search of the specified values on the Url and Uri fields An example value will be paloaltonetworks.com,Cortex XSOAR which will provide results like https://apps.paloaltonetworks.com and https://demisto.com | Optional |
```
Comment (yaakovi) [inline Packs/CortexDataLake/Integrations/CortexDataLake/README.md:896]: revert or update the integration yml too.
also, the description doesn't make sense after the change - please fix it.

File: Packs/AutoFocus/Integrations/AutofocusV2/README.md [hunk 3]
```diff
@@ -100,28 +93,7 @@ Copy the query value from the opening curly bracket `{` until the `,"scope"` par
...
-| AutoFocus.SamplesSearch.Artifact.m | Number | How many set the artifact as malicious. | 
-| AutoFocus.SamplesSearch.Artifact.confidence | String | How confident the decision. | 
-| AutoFocus.SamplesSearch.Artifact.indicator | String | The indicator that was tested. | 
-| AutoFocus.SamplesSearch.Artifact.indicator_type | String | The indicator type, for example: Mutex, User agent, IPv4, Domain. | 
-| File.Size | Number | The size of the file in bytes. | 
-| File.SHA1 | String | The SHA1 hash of the file. | 
-| File.SHA256 | String | The SHA256 hash of the file. | 
-| File.Type | String | The file type, as determined by libmagic (same as displayed in file entries). | 
-| File.Tags | String | The tags of the file. | 
+
```
Comment (yaakovi) [inline Packs/AutoFocus/Integrations/AutofocusV2/README.md:96]: why was this removed?

File: Packs/Lokpath_Keylight/.pack-ignore [hunk 4]
```diff
@@ -1,2 +1,5 @@
 [file:Lockpath_KeyLight_v2.yml]
 ignore=IN126,BA108,BA109
+
+[file:README.md]
+ignore=RM105
```
Comment (yaakovi) [inline Packs/Lokpath_Keylight/.pack-ignore:5]: why are we ignoring it here?

**Review by yaakovi — COMMENTED** [review]
sometimes you used `Cortex XSOAR` and sometimes just `XSOAR`.
we should be consistent.

when updating descriptions in the README files, we should update the relevant yml files too.

File: Packs/Lokpath_Keylight/.pack-ignore (hunk 4 above)
Comment (yaakovpraisler) [reply Packs/Lokpath_Keylight/.pack-ignore:5]: it's for examples, which I did not change

File: Packs/Lokpath_Keylight/.pack-ignore [hunk 5]
```diff
@@ -1,2 +1,5 @@
 [file:Lockpath_KeyLight_v2.yml]
-ignore=IN126,BA108,BA109
+ignore=IN126
```
Comment (roysagi) [inline Packs/Lokpath_Keylight/.pack-ignore:2]: why did you take out the BA errors?

**Review by roysagi — CHANGES_REQUESTED** [review]

File: Packs/DeprecatedContent/Integrations/PaloAltoNetworksCortex/README.md [hunk 6]
```diff
@@ -39,28 +39,28 @@
...
+<li><strong> <font style="vertical-align: inherit;"> <font style="vertical-align: inherit;">Authentication Token</font> </font> </strong> <font style="vertical-align: inherit;"> <font style="vertical-align: inherit;"> : received from the <a href="#h_ed302a66-2e1a-4748-abf9-3f771021c5f0" target="_self">Activate XSOAR on Palo Alto Networks Cortex Hub</a> procedure. </font> </font></li>
+<li><font style="vertical-align: inherit;"> <font style="vertical-align: inherit;"> <strong>Authentication ID</strong>:&nbsp;received from the <a href="#h_ed302a66-2e1a-4748-abf9-3f771021c5f0" target="_self">Activate XSOAR on Palo Alto Networks Cortex Hub</a> procedure. </font> </font></li>
+<li><font style="vertical-align: inherit;"> <font style="vertical-align: inherit;"> <strong>Authentication Key</strong>: received from the <a href="#h_ed302a66-2e1a-4748-abf9-3f771021c5f0" target="_self">Activate XSOAR on Palo Alto Networks Cortex Hub</a> procedure. </font> </font></li>
 </ul>
 </li>
-<li>Click&nbsp;<strong>Test</strong> to validate the integration and Demisto App Token.</li>
+<li>Click&nbsp;<strong>Test</strong> to validate the integration and XSOAR App Token.</li>
```
[the remark this answers is not in the document]
Comment (yaakovpraisler) [reply Packs/DeprecatedContent/Integrations/PaloAltoNetworksCortex/README.md:60]: done.

File: Packs/DeprecatedContent/Integrations/integration-SafeBreach_README.md [hunk 7]
```diff
@@ -17,41 +17,41 @@
...
 </tr>
 </tbody>
 </table>
 </li>
 </ol>
 <h3> </h3>
-<p>After you integrate Demisto, SafeBreach Management users can drill down in a simulation and use the <strong>Send To</strong> button to send the simulation results to Demisto. For more information see the Drilling Down for More about a Simulation article on the <a href="https://support.safebreach.com/hc/en-us">SafeBreach support site</a>.</p>
-<p><strong>NOTE</strong>: You can also use the <em>update</em> and <em>show</em> commands to change and view details about Demisto plugins.</p>
+<p>After you integrate Cortex XSOAR, SafeBreach Management users can drill down in a simulation and use the <strong>Send To</strong> button to send the simulation results to Cortex XSOAR. For more information see the Drilling Down for More about a Simulation article on the <a href="https://support.safebreach.com/hc/en-us">SafeBreach support site</a>.</p>
+<p><strong>NOTE</strong>: You can also use the <em>update</em> and <em>show</em> commands to change and view details about Cortex XSOAR plugins.</p>
```
[the remark this answers is not in the document]
Comment (yaakovpraisler) [reply Packs/DeprecatedContent/Integrations/integration-SafeBreach_README.md:54]: done.

File: Packs/Devo/Integrations/Devo_v2/README.md [hunk 8]
```diff
@@ -377,9 +377,9 @@ N/A
 
 
 #### Youtube Video Demo (Click Image, Will redirect to youtube)
-[![Devo-Demisto Plugin Demo](https://img.youtube.com/vi/jyUqEcWOXfU/0.jpg)](https://www.youtube.com/watch?v=jyUqEcWOXfU "Devo Demisto Demo")
+[![Devo-Demisto Plugin Demo](https://img.youtube.com/vi/jyUqEcWOXfU/0.jpg)](https://www.youtube.com/watch?v=jyUqEcWOXfU "Devo Cortex XSOAR Demo")
```
[the remark this answers is not in the document]
Comment (yaakovpraisler) [reply Packs/Devo/Integrations/Devo_v2/README.md:380]: done.

File: Packs/EDL/Playbooks/playbook-PAN-OS_EDL_Service_Configuration_README.md [hunk 9]
```diff
@@ -29,8 +29,8 @@ This playbook uses the following sub-playbooks, integrations, and scripts.
 
 | **Name** | **Description** | **Default Value** | **Required** |
 | --- | --- | --- | --- |
-| ip-edl-object-name | Set a name for the EDL object that will be configured on pan\-os. This value is used to set the rule name as well. | Demisto Remediation - IP EDL | Optional |
-| url-edl-object-name | Set a name for the EDL object that will be configured on pan\-os. This value is used to set the rule name as well. | Demisto Remediation - URL EDL | Optional |
+| ip-edl-object-name | Set a name for the EDL object that will be configured on pan\-os. This value is used to set the rule name as well. | XSOAR Remediation - IP EDL | Optional |
+| url-edl-object-name | Set a name for the EDL object that will be configured on pan\-os. This value is used to set the rule name as well. | XSOAR Remediation - URL EDL | Optional |
```
[the remark this answers is not in the document]
Comment (yaakovpraisler) [reply Packs/EDL/Playbooks/playbook-PAN-OS_EDL_Service_Configuration_README.md:33]: done.

File: Packs/EWS/Integrations/EWSO365/EWSO365_description.md [hunk 10]
```diff
@@ -1,5 +1,5 @@
-To allow access to EWS O365, an administrator has to approve the Demisto app using an admin consent flow, by clicking on the following [link](https://oproxy.demisto.ninja/ms-ews-o365).
-After authorizing the Demisto app, you will get an ID, Token, and Key, which needs to be added to the integration instance configuration's corresponding fields.
+To allow access to EWS O365, an administrator has to approve the Cortex XSOAR app using an admin consent flow, by clicking on the following [link](https://oproxy.demisto.ninja/ms-ews-o365).
+After authorizing the Cortex XSOAR app, you will get an ID, Token, and Key, which needs to be added to the integration instance configuration's corresponding fields.
```
[the remark this answers is not in the document]
Comment (yaakovpraisler) [reply Packs/EWS/Integrations/EWSO365/EWSO365_description.md:2]: done.

File: Packs/ExtraHop/Integrations/ExtraHop_v2/README.md [hunk 11]
```diff
@@ -12,12 +12,12 @@
...
 <li>Enable guided investigation and response through playbooks and automation scripts.</li>
-<li>Interrogate the ExtraHop Reveal(x) REST API using the simple and powerful Demisto CLI.</li>
+<li>Interrogate the ExtraHop Reveal(x) REST API using the simple and powerful Cortex XSOAR CLI.</li>
 </ul><h2>Detailed Description</h2>
-<p>Visit the <a href="https://www.extrahop.com/customers/community/bundles/extrahop/demisto-integration/">ExtraHop + Demisto Setup Guide</a> for detailed integration instructions.</p>
+<p>Visit the <a href="https://www.extrahop.com/customers/community/bundles/extrahop/demisto-integration/">ExtraHop + XSOAR Setup Guide</a> for detailed integration instructions.</p>
 <h2>Fetch Incidents</h2>
-<p>Incidents are pushed in via the Demisto REST API by a trigger running on the ExtraHop Reveal(x) appliance.</p>
-<h2>Configure ExtraHop Reveal(x) on Demisto</h2>
+<p>Incidents are pushed in via the XSOAR REST API by a trigger running on the ExtraHop Reveal(x) appliance.</p>
```
[the remark this answers is not in the document]
Comment (yaakovpraisler) [reply Packs/ExtraHop/Integrations/ExtraHop_v2/README.md:19]: done.

File: Packs/GmailSingleUser/Integrations/GmailSingleUser/README.md [hunk 12]
```diff
@@ -4,17 +4,17 @@ Use the Gmail Single User integration to send emails and fetch emails as inciden
 
 ## Application Authorization Flow
 
-To allow Cortex XSOAR to access Gmail, the user has to approve the Demisto App using an OAuth 2.0 authorization flow. Follow these steps to authorize the Demisto app in Gmail.
+To allow Cortex XSOAR to access Gmail, the user has to approve the Cortex XSOAR App using an OAuth 2.0 authorization flow. Follow these steps to authorize the Cortex XSOAR app in Gmail.
```
[the remark this answers is not in the document]
Comment (yaakovpraisler) [reply Packs/GmailSingleUser/Integrations/GmailSingleUser/README.md:7]: done.

File: Packs/GmailSingleUser/Integrations/GmailSingleUser/README.md [hunk 13]
```diff
@@ -4,17 +4,17 @@ Use the Gmail Single User integration to send emails and fetch emails as inciden
...
 
 1. Create and save an integration instance of the Gmail Single User integration. Do not fill in the *Auth Code* field, this will be obtained in the next steps.
 2. To obtain the **Auth Code** run the following command in the playground: ***!gmail-auth-link***. Access the link you receive to authenticate your Gmail account. 
 3. Complete the authentication process and copy the received code to the **Auth Code** configuration parameter of the integration instance. 
 4. Save the instance.
 5. To verify that authentication was configured correctly, run the ***!gmail-auth-test***.
 
-**NOTE:** The Demisto App is going through the Google verification process. During the verification process the app is not fully verified, and you may receive from Google an "unverified app" warning in the authorization flow.
+**NOTE:** The Cortex XSOAR App is going through the Google verification process. During the verification process the app is not fully verified, and you may receive from Google an "unverified app" warning in the authorization flow.
```
[the remark this answers is not in the document]
Comment (yaakovpraisler) [reply Packs/GmailSingleUser/Integrations/GmailSingleUser/README.md:15]: done.

File: Packs/GmailSingleUser/Integrations/GmailSingleUser/README.md [hunk 14]
```diff
@@ -4,17 +4,17 @@ Use the Gmail Single User integration to send emails and fetch emails as inciden
...
 
-**GSuite Admins:** You can choose to trust the Demisto App so your users can configure the App. Instructions:
+**GSuite Admins:** You can choose to trust the Cortex XSOAR App so your users can configure the App. Instructions:
```
[the remark this answers is not in the document]
Comment (yaakovpraisler) [reply Packs/GmailSingleUser/Integrations/GmailSingleUser/README.md:17]: done.

File: Packs/GmailSingleUser/Integrations/GmailSingleUser/README.md [hunk 15]
```diff
@@ -25,7 +25,7 @@ To allow Cortex XSOAR to access Gmail, the user has to approve the Demisto App u
 
 Additional info available at: https://support.google.com/a/answer/7281227
 
-**Optional:** You can use your own Google App instead of the default Demisto App. To create your own app, follow the [Google instructions for Desktop Apps](https://developers.google.com/identity/protocols/OAuth2InstalledApp#prerequisites). 
+**Optional:** You can use your own Google App instead of the default Cortex XSOAR App. To create your own app, follow the [Google instructions for Desktop Apps](https://developers.google.com/identity/protocols/OAuth2InstalledApp#prerequisites).
```
[the remark this answers is not in the document]
Comment (yaakovpraisler) [reply Packs/GmailSingleUser/Integrations/GmailSingleUser/README.md:28]: done.

File: Packs/Lokpath_Keylight/.pack-ignore (hunk 5 above)
Comment (yaakovpraisler) [reply Packs/Lokpath_Keylight/.pack-ignore:2]: mistake, fixed.

File: Packs/AzureCompute/Integrations/AzureCompute_v2/AzureCompute_v2_description.md [hunk 16]
```diff
@@ -7,6 +7,6 @@ There are 2 application authentication methods available:
 
 Depending on the authentication method that you use, the integration parameters might change.
 
-To allow us access to Azure Compute, an administrator has to approve the Demisto app using an admin consent flow, by clicking [here](https://oproxy.demisto.ninja/ms-azure-compute).
-After authorizing the Demisto app, you will receive an ID, Token, and Key, which needs to be added to the integration instance configuration's corresponding fields. After giving consent, the application must have a role assigned, so it can access the relevant resources per subscription. 
+To allow us access to Azure Compute, an administrator has to approve the Cortex XSOAR app using an admin consent flow, by clicking [here](https://oproxy.demisto.ninja/ms-azure-compute).
+After authorizing the Cortex XSOAR app, you will receive an ID, Token, and Key, which needs to be added to the integration instance configuration's corresponding fields. After giving consent, the application must have a role assigned, so it can access the relevant resources per subscription.
```
[the remark this answers is not in the document]
Comment (yaakovpraisler) [reply Packs/AzureCompute/Integrations/AzureCompute_v2/AzureCompute_v2_description.md:11]: done.

**Review by roysagi — APPROVED** [review]
